### PR TITLE
Add experimental high contrast mode

### PR DESCRIPTION
**Pull request by sdorra**
## Proposed changes

Add an experimental high contrast color theme to SCM-Manager. The high contrast mode uses a dark background and color with a high contrast for a better accessibility.
The change adds the theme to ui-styles and theme switcher to the storybook of ui-components.

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [X] PR is well described and the description can be used as a commit message on squash
- [X] Related issues linked to PR if existing and labels set
- [X] New ui components are tested inside the storybook (module ui-components only) 
- [X] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog` or CHANGELOG.md is updated for plugins

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

**Review by eheimbuch — APPROVED** [review]

File: scm-ui/ui-webapp/public/locales/de/commons.json [hunk 1]
```diff
@@ -94,7 +94,23 @@
     "error-title": "Fehler",
     "error-subtitle": "Das Profil kann nicht angezeigt werden.",
     "error": "Fehler",
-    "error-message": "'me' ist nicht definiert"
+    "error-message": "'me' ist nicht definiert",
+    "theme": {
+      "nav": {
+        "label": "Theme",
+        "title": "Wähle dein Theme"
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/de/commons.json:101]:
```suggestion
        "title": "Wähle dein Design"
```

File: scm-ui/ui-webapp/public/locales/de/commons.json [hunk 2]
```diff
@@ -94,7 +94,23 @@
...
+      },
+      "subtitle": "Wähle dein Theme",
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/de/commons.json:103]:
```suggestion
      "subtitle": "Wähle dein Design",
```

File: scm-ui/ui-webapp/public/locales/de/commons.json [hunk 3]
```diff
@@ -94,7 +94,23 @@
...
+      "submit": "Anwenden",
+      "light": {
+        "displayName": "Light",
+        "description": "Light is das Standard Thema des SCM-Managers"
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/de/commons.json:107]:
```suggestion
        "displayName": "Hell",
        "description": "'Hell' is das Standard-Design des SCM-Managers"
```

File: scm-ui/ui-webapp/public/locales/de/commons.json [hunk 4]
```diff
@@ -94,7 +94,23 @@
...
+      },
+      "highcontrast": {
+        "displayName": "High contrast",
+        "description": "High contrast ist ein dunkles Thema mit einem hohen Kontrast"
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/de/commons.json:111]:
```suggestion
        "displayName": "Hoher Kontrast",
        "description": "'Hoher Kontrast' ist ein dunkles Design mit einem hohen Kontrast"
```

File: scm-ui/ui-webapp/public/locales/en/commons.json [hunk 5]
```diff
@@ -95,7 +95,23 @@
     "error-title": "Error",
     "error-subtitle": "Cannot display profile",
     "error": "Error",
-    "error-message": "'me' is undefined"
+    "error-message": "'me' is undefined",
+    "theme": {
+      "nav": {
+        "label": "Theme",
+        "title": "Choose your theme"
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/en/commons.json:102]:
```suggestion
        "title": "Choose your Theme"
```

File: scm-ui/ui-webapp/public/locales/en/commons.json [hunk 6]
```diff
@@ -95,7 +95,23 @@
...
+      },
+      "subtitle": "Choose your theme",
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/en/commons.json:104]:
```suggestion
      "subtitle": "Choose your Theme",
```

**Review by pfeuffer — CHANGES_REQUESTED** [review]
Just some minor suggestions and one typo in the german frontend translation.

File: scm-ui/ui-webapp/public/locales/de/commons.json [hunk 7]
```diff
@@ -94,7 +94,23 @@
     "error-title": "Fehler",
     "error-subtitle": "Das Profil kann nicht angezeigt werden.",
     "error": "Fehler",
-    "error-message": "'me' ist nicht definiert"
+    "error-message": "'me' ist nicht definiert",
+    "theme": {
+      "nav": {
+        "label": "Theme",
```
Comment (pfeuffer) [inline scm-ui/ui-webapp/public/locales/de/commons.json:100]:
```suggestion
        "label": "Design",
```

**Pull request merged**